private/model/api/operation.go

```diff
@@ -71,7 +73,11 @@ const op{{ .ExportedName }} = "{{ .Name }}"
 //    if err == nil { // resp is now filled
 //        fmt.Println(resp)
 //    }
+{{ $crosslinkURL := GetCrosslinkURL $.API.BaseCrosslinkURL $.API.APIName $.API.Metadata.UID $.ExportedName -}}
```

Comment:
are the `$.` needed? generally we haven't used that notation for the templates before. Unless its needed we probably want to be consistent.